README.md

```diff
@@ -77,11 +77,11 @@ As such, we encourage other scientific tool builders in academia or industry to
 this project. All code is freely available for reuse under the [MIT license](https://opensource.org/licenses/MIT).
 
 
-Before extending cellxgene, we encourage you to reach out to us with ideas or questions. It might be possible that an 
+Before extending CELLxGENE, we encourage you to reach out to us with ideas or questions. It might be possible that an 
```

Comment:
```suggestion
Before extending CELLxGENE Annotate, we encourage you to reach out to us with ideas or questions. It might be possible that an 
```

Per comments on other PRs, we're trying to clarify that CELLxGENE is now the brand family and the desktop application formerly known as cellxgene is now CELLxGENE Annotate. So we should replace all "cellxgene" references to the desktop app with CZ CELLxGENE Annotate, CELLxGENE Annotate, or Annotate. :+1: